test/remote/gateways/remote_cyber_source_rest_test.rb

```diff
@@ -70,6 +98,32 @@ def test_successful_purchase
     assert_nil response.params['_links']['capture']
   end
 
+  def test_successful_authorize_with_apple_pay
+    response = @gateway.authorize(@amount, @apple_pay, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'AUTHORIZED', response.message
+    refute_empty response.params['_links']['capture']
+  end
+
+  def test_successful_authorize_with_google_pay
+    response = @gateway.authorize(@amount, @apple_pay, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'AUTHORIZED', response.message
+    refute_empty response.params['_links']['capture']
+  end
+
+  def test_successful_purchase_with_google_pay_jcb
+    response = @gateway.purchase(@amount, @apple_pay_jcb, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'AUTHORIZED', response.message
+  end
+
```

Comment:
💬 Comment

Considering the code for the `consumerAuthenticationInformation` key maybe is a good idea to test other card brands like amex, discover, etc.

Reply:
I used amex and discover, and the gateway approved those transactions.